pom.xml

```diff
@@ -420,6 +421,21 @@ SPDX-License-Identifier: MIT
       <groupId>org.hsqldb</groupId>
       <artifactId>hsqldb</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.sentry</groupId>
+      <artifactId>sentry-spring-boot-starter</artifactId>
+      <version>${sentry.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.sentry</groupId>
```

Comment:
Ik neem normaal niet de transitieve dependencies op van de "spring-boot-starter"  modules, maar alleen de starter modules en neem dan onder de `banTransitiveDependencies\excludes` een exclusion op; 

https://github.com/B3Partners/tailormap-api/blob/cd52128a145c1e41ebb3b38c4a48e88bd887996d/pom.xml#L1101-L1134

dan blijft het in de toekomst met een upgrade van de starter module werken mocht daar een dependency bij komen.

Reply:
Aha! Had helemaal gemist dat ik beiden de `sentry-spring-boot-starter` en `sentry` artifacts had..